src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/RFMPagesStepDef.scala

```diff
@@ -79,31 +79,37 @@ class RFMPagesStepDef extends BaseStepDef with BrowserDriver {
         Wait.waitForTagNameToBeRefreshed("h1")
         Wait.waitForElementToPresentByCssSelector(RFMEnterPillar2IdPage.pillar2topuptaxid)
         Input.sendKeysByCss(name, RFMEnterPillar2IdPage.pillar2topuptaxid)
+        RFMStartPage.clickContinue()
 
       case "contact name" =>
         Wait.waitForTagNameToBeRefreshed("h1")
         Wait.waitForElementToPresentByCssSelector(RFMContactDetailNamePage.nameField)
         Input.sendKeysByCss(name, RFMContactDetailNamePage.nameField)
+        RFMStartPage.clickContinue()
 
       case "contact email" =>
         Wait.waitForTagNameToBeRefreshed("h1")
         Wait.waitForElementToPresentByCssSelector(RFMContactEmailPage.emailField)
         Input.sendKeysByCss(name, RFMContactEmailPage.emailField)
+        RFMStartPage.clickContinue()
 
       case "contact number" =>
         Wait.waitForTagNameToBeRefreshed("h1")
         Wait.waitForElementToPresentByCssSelector(RFMContactInputPage.telephoneField)
         Input.sendKeysByCss(name, RFMContactInputPage.telephoneField)
+        RFMStartPage.clickContinue()
 
       case "New NFM Name" =>
         Wait.waitForTagNameToBeRefreshed("h1")
         Wait.waitForElementToPresentByCssSelector(RFMNewNFMContactNamePage.nameField)
         Input.sendKeysByCss(name, RFMNewNFMContactNamePage.nameField)
+        RFMStartPage.clickContinue()
 
       case "second contact number" =>
         Wait.waitForTagNameToBeRefreshed("h1")
         Wait.waitForElementToPresentByCssSelector(RFMSecondContactTelephonePage.telephoneField)
         Input.sendKeysByCss(name, RFMSecondContactTelephonePage.telephoneField)
+        RFMStartPage.clickContinue()
```

Comment:
as this is same logic for all the switch cases , we can add it after switch block

Reply:
Done